omnibus/config/software/datadog-logs-agent.rb

```diff
@@ -0,0 +1,20 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https:#www.datadoghq.com/).
+# Copyright 2017 Datadog, Inc.
+
+name "datadog-logs-agent"
+always_build true
+
+logs_agent_version = ENV['logs_agent_version']
```

Comment:
Just an headsup, we're trying to remove any env var access from the software definitions to have more predictable builds (omnibus has its own way to inject a version from the outside).